Packs/Jira/Integrations/JiraV3/JiraV3.yml

```diff
@@ -498,6 +503,63 @@ script:
     - contextPath: Ticket.LastUpdate
       description: The last time the ticket was updated.
       type: Date
+  - arguments:
+    - description: Deprecated. Please use issue_id or issue_key.
```

Comment:
It is actually required, since this argument is already used in JiraV2, and playbooks can be using the **issueId** argument, upgrading to JiraV3 can break their playbooks if we remove this argument, therefore, it is required to keep it, and mark it as hidden. We map JiraV2 arguments to JiraV3 in the code, using the **map_v2_args_to_v3** function